From f894635125fbef79206486223a65fa385453e75f Mon Sep 17 00:00:00 2001
From: Vincent Rabaud <vrabaud@willowgarage.com>
Date: Wed, 13 Mar 2013 11:08:16 -0700
Subject: [PATCH 1/2] Customized debian rules file

---
 debian/rules.em |   25 +++++++++++++++++++++----
 1 file changed, 21 insertions(+), 4 deletions(-)

diff --git a/debian/rules.em b/debian/rules.em
index d45dd5f..339bbe1 100644
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -16,6 +16,22 @@ export DH_OPTIONS=-v --buildsystem=cmake
 export LDFLAGS=
 export PKG_CONFIG_PATH=@(InstallationPrefix)/lib/pkgconfig
 
+CMAKE_FLAGS = \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DENABLE_SSE=ON \
+        -DENABLE_SSE2=ON \
+        -DENABLE_SSE3=ON \
+        -DENABLE_SSE42=OFF \
+        -DENABLE_SSE41=OFF \
+        -DENABLE_FAST_MATH=OFF \
+        -DBUILD_EXAMPLES=OFF \
+        -DBUILD_DOCS=ON \
+        -DBUILD_PERF_TESTS=OFF \
+        -DBUILD_TESTS=OFF \
+        -DBUILD_NEW_PYTHON_SUPPORT=ON \
+        -DWITH_V4L=ON \
+        -DWITH_CUDA=OFF \
+
 %:
 	dh  $@@
 
@@ -23,11 +39,12 @@ override_dh_auto_configure:
 	# In case we're installing to a non-standard location, look for a setup.sh
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
-	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
-	dh_auto_configure -- \
+	if [ -f "/opt/ros/groovy/setup.sh" ]; then . "/opt/ros/groovy/setup.sh"; fi && \
+	dh_auto_configure -Scmake -- \
+		-DCMAKE_INSTALL_PREFIX="/opt/ros/groovy" \
+		-DCMAKE_PREFIX_PATH="/opt/ros/groovy" \
 		-DCATKIN_BUILD_BINARY_PACKAGE="1" \
-		-DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
-		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)"
+		$(CMAKE_FLAGS)
 
 override_dh_auto_build:
 	# In case we're installing to a non-standard location, look for a setup.sh
-- 
1.7.9.5

