From 3263a274df3a157ef546f5e5772300481d086b83 Mon Sep 17 00:00:00 2001
From: Vincent Rabaud <vincent.rabaud@gmail.com>
Date: Fri, 13 Jun 2014 18:39:19 +0200
Subject: [PATCH] Customized debian rules file

---
 debian/rules.em | 28 +++++++++++++++++++++++-----
 1 file changed, 23 insertions(+), 5 deletions(-)

diff --git a/debian/rules.em b/debian/rules.em
index 06777bd..dade043 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -16,6 +16,23 @@ export DH_OPTIONS=-v --buildsystem=cmake
 export LDFLAGS=
 export PKG_CONFIG_PATH=@(InstallationPrefix)/lib/pkgconfig
 
+
+CMAKE_FLAGS = \
+	-DCMAKE_BUILD_TYPE=RelWithDebInfo \
+	-DENABLE_SSE=ON \
+	-DENABLE_SSE2=ON \
+	-DENABLE_SSE3=ON \
+	-DENABLE_SSE42=OFF \
+	-DENABLE_SSE41=OFF \
+	-DENABLE_FAST_MATH=OFF \
+	-DBUILD_EXAMPLES=OFF \
+	-DBUILD_DOCS=OFF \
+	-DBUILD_PERF_TESTS=OFF \
+	-DBUILD_TESTS=OFF \
+	-DBUILD_NEW_PYTHON_SUPPORT=ON \
+	-DWITH_V4L=ON \
+	-DWITH_CUDA=OFF
+
 %:
 	dh  $@@
 
@@ -23,11 +40,12 @@ override_dh_auto_configure:
 	# In case we're installing to a non-standard location, look for a setup.sh
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
-	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
-	dh_auto_configure -- \
-		-DCATKIN_BUILD_BINARY_PACKAGE="1" \
-		-DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
-		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)"
+	if [ -f "/opt/ros/groovy/setup.sh" ]; then . "/opt/ros/groovy/setup.sh"; fi && \
+	dh_auto_configure -Scmake -- \
+		-DCMAKE_INSTALL_PREFIX="/opt/ros/groovy" \
+		-DCMAKE_PREFIX_PATH="/opt/ros/groovy" \
+ 		-DCATKIN_BUILD_BINARY_PACKAGE="1" \
+		$(CMAKE_FLAGS)
 
 override_dh_auto_build:
 	# In case we're installing to a non-standard location, look for a setup.sh
-- 
1.9.1

