From 684c0e25007e6b2a8b07a21e3d8ab948ce7db626 Mon Sep 17 00:00:00 2001
From: Vincent Rabaud <vincent.rabaud@gmail.com>
Date: Fri, 13 Jun 2014 18:32:22 +0200
Subject: [PATCH 1/5] Added install rule for package.xml

---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b610ecf..8eac514 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1059,3 +1059,6 @@ endif()
 # ----------------------------------------------------------------------------
 
 include(cmake/OpenCVPackaging.cmake)
+
+# Install catkin package.xml
+install(FILES package.xml DESTINATION share/opencv2)
-- 
1.9.1

