From fa84d7ec964fbad76772e56905ca39847ce1d039 Mon Sep 17 00:00:00 2001
From: Vincent Rabaud <vincent.rabaud@gmail.com>
Date: Tue, 16 Jul 2013 11:03:06 +0000
Subject: [PATCH 2/2] install to hydro ........

---
 debian/rules.em | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/debian/rules.em b/debian/rules.em
index 339bbe1..37957ce 100644
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -39,10 +39,10 @@ override_dh_auto_configure:
 	# In case we're installing to a non-standard location, look for a setup.sh
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
-	if [ -f "/opt/ros/groovy/setup.sh" ]; then . "/opt/ros/groovy/setup.sh"; fi && \
+	if [ -f "/opt/ros/hydro/setup.sh" ]; then . "/opt/ros/hydro/setup.sh"; fi && \
 	dh_auto_configure -Scmake -- \
-		-DCMAKE_INSTALL_PREFIX="/opt/ros/groovy" \
-		-DCMAKE_PREFIX_PATH="/opt/ros/groovy" \
+		-DCMAKE_INSTALL_PREFIX="/opt/ros/hydro" \
+		-DCMAKE_PREFIX_PATH="/opt/ros/hydro" \
 		-DCATKIN_BUILD_BINARY_PACKAGE="1" \
 		$(CMAKE_FLAGS)
 
-- 
1.8.3.4 (Apple Git-47)

