From dac3808cbfcf65ac167e430a82513dfbb413ceda Mon Sep 17 00:00:00 2001
From: Vincent Rabaud <vrabaud@willowgarage.com>
Date: Mon, 21 Oct 2013 00:03:44 +0200
Subject: [PATCH 2/2] remove some optimizations for the 32-bit bug

---
 debian/rules.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 339bbe1..a8e6729 100644
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -17,7 +17,7 @@ export LDFLAGS=
 export PKG_CONFIG_PATH=@(InstallationPrefix)/lib/pkgconfig
 
 CMAKE_FLAGS = \
-        -DCMAKE_BUILD_TYPE=Release \
+        -DCMAKE_BUILD_TYPE=RelWithDebInfo \
         -DENABLE_SSE=ON \
         -DENABLE_SSE2=ON \
         -DENABLE_SSE3=ON \
-- 
1.8.1.2

