From 46496a3faec760d3e2218832f3e143ce8c2b4ac5 Mon Sep 17 00:00:00 2001
From: Vincent Rabaud <vrabaud@willowgarage.com>
Date: Thu, 7 Nov 2013 23:19:28 +0100
Subject: [PATCH 4/4] disable building the docs

---
 debian/rules.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index aa5931b..86f274d 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -25,7 +25,7 @@ CMAKE_FLAGS = \
         -DENABLE_SSE41=OFF \
         -DENABLE_FAST_MATH=OFF \
         -DBUILD_EXAMPLES=OFF \
-        -DBUILD_DOCS=ON \
+        -DBUILD_DOCS=OFF \
         -DBUILD_PERF_TESTS=OFF \
         -DBUILD_TESTS=OFF \
         -DBUILD_NEW_PYTHON_SUPPORT=ON \
-- 
1.8.1.2

