From a9f3cb428437a02dce37b5840ba56e4a8711a33f Mon Sep 17 00:00:00 2001
From: Vincent Rabaud <vrabaud@willowgarage.com>
Date: Thu, 28 Feb 2013 21:48:42 -0800
Subject: [PATCH 6/6] install opencv2.pc

---
 cmake/OpenCVGenPkgconfig.cmake |    1 +
 1 file changed, 1 insertion(+)

diff --git a/cmake/OpenCVGenPkgconfig.cmake b/cmake/OpenCVGenPkgconfig.cmake
index 6a11e08..3661a8a 100644
--- a/cmake/OpenCVGenPkgconfig.cmake
+++ b/cmake/OpenCVGenPkgconfig.cmake
@@ -73,4 +73,5 @@ configure_file("${OpenCV_SOURCE_DIR}/cmake/templates/opencv-XXX.pc.cmake.in" "${
 
 if(UNIX AND NOT ANDROID)
   install(FILES ${CMAKE_BINARY_DIR}/unix-install/${OPENCV_PC_FILE_NAME} DESTINATION ${OPENCV_LIB_INSTALL_PATH}/pkgconfig)
+  install(FILES ${CMAKE_BINARY_DIR}/unix-install/opencv2.pc DESTINATION ${OPENCV_LIB_INSTALL_PATH}/pkgconfig)
 endif()
-- 
1.7.9.5

